Packs/Devo/Integrations/Devo_v2/Devo_v2.py

```diff
@@ -916,12 +917,6 @@ def main():
                 total = total + COUNT_SINGLE_TABLE
                 demisto.results(run_query_command(OFFSET, items_per_page))
         elif demisto.command() == "devo-get-alerts":
-            from_date = int(demisto.args().get("from"))
```

Comment:
Can you please explain, are you using the arguments "from" and "to" anywhere else?
Because it is in the yml but now has no use in the python. is that correct?

Reply:
Yes, it has been used in functions like run_query_command , get_alerts_command, multi_table_query_command